USAGE.md

```diff
@@ -0,0 +1,65 @@
+[[Back to main IRS README](README.md)]
```

Comment:
IMHO we should use Github to navigate through the project.

Reply:
see my comment above